terminusdb_client/woqlview/woql_view.py

```diff
@@ -1,3 +1,9 @@
+import sys
+if sys.version_info[0]==3 and sys.version_info[1]>7:
+    from __future__ import annotations
```

Comment:
for now, let's add "# noqa" at the end of this like to by-pass the flaks8 check. See if Travis pass this time.